tests/product/base_product_case.py

```diff
@@ -335,16 +335,23 @@ def assert_installed(self, container, cluster=None, msg=None):
             cluster = self.cluster
         try:
             check_rpm = cluster.exec_cmd_on_host(
-                container, 'rpm -q presto')
-            self.assertEqual(self.presto_rpm_filename[:-4] + '\n', check_rpm,
-                             msg=msg)
+                container, 'rpm -q presto-server-rpm')
+            self.assertRegexpMatches(
+                check_rpm, self.presto_rpm_filename[:-4] + r'.*\n', msg=msg
```

Comment:
I understand the goal, I just think the construction of the regex is a bit contrived.  The rpm name isn't actually tied to the filename at all.  It happens to relate, but you made up the name of the file when you downloaded it.  Maybe put 'presto-server-rpm.*' as its own constant or build them both from a constant that's just 'presto-server-rpm'?